pjmedia/include/pjmedia/codec.h

```diff
@@ -722,6 +722,9 @@ typedef struct pjmedia_codec_mgr
     /** Array of codec descriptor. */
     struct pjmedia_codec_desc    codec_desc[PJMEDIA_CODEC_MGR_MAX_CODECS];
 
+    /** Array of codec id. */
+    pj_str_t                     codec_list[PJMEDIA_CODEC_MGR_MAX_CODECS];
```

Comment:
since codec desc can be removed, make sure to handle the case that the codec_desc and codec_list have different number of elements